src/Confluent.SchemaRegistry.Serdes.Json/JsonSerializer.cs

```diff
@@ -53,7 +53,7 @@ namespace Confluent.SchemaRegistry.Serdes
     ///     integration of System.Text.Json and JSON Schema, so this
     ///     is not yet supported by the serializer.
     /// </remarks>
-    public class JsonSerializer<T> : IAsyncSerializer<T>  where T : new()
```

Comment:
Should still perhaps be class constraint?

Reply:
Fair point, change has been made